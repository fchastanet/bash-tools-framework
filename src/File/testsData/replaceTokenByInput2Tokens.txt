stuff before
@token@
stuff after 1
@token@
stuff after 2
