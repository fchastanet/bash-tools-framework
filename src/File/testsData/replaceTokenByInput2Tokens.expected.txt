stuff before
replaceTokenByInput injected
multi lines
stuff after 1
replaceTokenByInput injected
multi lines
stuff after 2
